recipes/monit.rb

```diff
@@ -0,0 +1,39 @@
+#
```

Comment:
`monit` is clearly not something I want to manage in a community CK.
It depend on your own way to do (and can be different for other guy).

You can manage it in a wrapper CK or on a CK `twemproxy-monit` for example (who can use this one).